### PR TITLE
Diff with ignress analysis tests

### PR DESCRIPTION
**Pull request by shireenf-ibm**
new diff tests with ingress objects, changes descriptions in commit messages

### CODE REVIEW

File: tests/k8s_ingress_test_new/diff_output_from_k8s_ingress_test.csv [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+default/reviews-v1-545db77b95[ReplicaSet],default/productpage-v1-6b746f74dc[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v1-545db77b95[ReplicaSet],default/ratings-v1-b6994bb9[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v2-7bf8c9648f[ReplicaSet],default/productpage-v1-6b746f74dc[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v2-7bf8c9648f[ReplicaSet],default/ratings-v1-b6994bb9[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v3-84779c7bbc[ReplicaSet],default/productpage-v1-6b746f74dc[ReplicaSet],All Connections,TCP 9080,changed
+default/reviews-v3-84779c7bbc[ReplicaSet],default/ratings-v1-b6994bb9[ReplicaSet],All Connections,TCP 9080,changed
+0.0.0.0-255.255.255.255,default/unicorn[Deployment],No Connections,All Connections,added
+default/unicorn[Deployment],0.0.0.0-255.255.255.255,No Connections,All Connections,added
+default/unicorn[Deployment],default/details-v1-79f774bdb9[ReplicaSet],No Connections,TCP 9080,added
+{ingress-controller},default/unicorn[Deployment],No Connections,TCP 8080,added
```
Comment (adisos) [inline tests/k8s_ingress_test_new/diff_output_from_k8s_ingress_test.csv:11]: let's place differences with `{ingress-controller}` at the end, similar to connlist

File: pkg/netpol/diff/diff_test.go
Comment (adisos) [inline pkg/netpol/diff/diff_test.go:1]: - do we have a test with ingress/route resource existing in one dir but not in the other ?
- it is difficult to follow what these tests include, please specify in comment next to each test details on what it checks, such as "changed several netpols (specify their names) " , "added netpols .. " , "added Route ..." , "changed Ingress ..." , "added workload ..." , etc..

File: tests/k8s_ingress_test_new/diff_output_from_k8s_ingress_test.csv (hunk 1 above)
Comment (shireenf-ibm) [reply tests/k8s_ingress_test_new/diff_output_from_k8s_ingress_test.csv]: done

File: pkg/netpol/diff/diff_test.go
Comment (shireenf-ibm) [reply pkg/netpol/diff/diff_test.go:1]: done both (new tests + tests descriptions)

**Comment by shireenf-ibm** [comment]
closing, as it was merged into #202

**Pull request closed without merging**